test/api/ticket/api_test.go

```diff
@@ -0,0 +1,154 @@
+package ticket
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+func TestTicketCRUD(t *testing.T) {
+	for _, r := range Samples {
+		t.Run("Ticket "+r.Kind+" CRUD", func(t *testing.T) {
+
+			// Create a sample Application for the ticket.
+			app := api.Application{
+				Name: r.Application.Name,
+			}
+			assert.Must(t, Application.Create(&app))
+
+			// Create a sample identity for the tracker
+			identity := api.Identity{
+				Name: "Sample Identity",
+				Kind: "Sample Kind",
+			}
+			assert.Must(t, Identity.Create(&identity))
+
+			// create a sample tracker for the ticket.
+			tracker := api.Tracker{
```

Comment:
The Tracker fields might be better re-used from from Tracker API test Samples instead of writing it here as well as in List test.

Reply:
Shall i run a loop for tracker samples ?

Reply:
or shall i use only one tracker from the sample?

Reply:
One tracker from sample should be enough!